Gruvo/Gruvo/ClientApp/src/app/feed/feed.component.js

```diff
@@ -0,0 +1,3 @@
+"use strict";
```

Comment:
this type file (js and maps) shouldn't  tracked in git. I am surprised that it is here, is it because of your IDEs?